Cargo.toml

```diff
@@ -12,7 +12,7 @@ primitive-types = { version = "0.10", features = ["fp-conversion"], optional = t
 serde = { version = "1.0", features = ["derive"] }
 serde_json = "1.0"
 serde_with = "1.6"
-tokio = { version = "1.9", features = ["sync", "time"], optional = true }
+tokio = { version = "1.9", features = ["sync", "time", "rt"], optional = true }
```

Comment:
Is there anything in this PR which requires the `rt` feature or is it just generally good to have?

Reply:
Agree that we shouldn't need to add it if we don't use it.

Reply:
I forgot to include it in the description. The feature is needed because this module spawns a task. We should have been using the feature before.